content/en/blog/_posts/2020-09-03-warnings/index.md

```diff
@@ -320,6 +320,19 @@ If you want to be sure you notice deprecations as soon as possible and get a jum
 This could be used in a CI job to apply manifests to a current server,
 and required to pass with a zero exit code in order for the CI job to succeed.
 
+## Validating Admission Policy
+
+_Added in 2023_
+
+We introduced [Validating Admission Policy](/docs/reference/access-authn-authz/validating-admission-policy) in 1.26 as an alpha feature.
+Starting in Kubernetes v1.27, a field `validationAction` is added in ValidatingAdmissionPolicyBinding which is used to declare
+how Validations of the referenced ValidatingAdmissionPolicy are enforced. When it sets to `Warn`, a validation failure is reported to the request client
+in HTTP Warning headers, with a warning code of 299.
+
+<img alt="kubectl applying a manifest file when there is an existing a ValidatingAdmissionPolicy and a validationAction set to Warn, displaying a warning message and exiting with a non-zero exit code."
+src="kubectl-warnings-vap.png"
+style="width:637px;max-width:100%;">
```

Comment:
This isn't rendering - see https://deploy-preview-41332--kubernetes-io-main-staging.netlify.app/blog/2020/09/03/warnings/

Please fix that and check that the [local preview](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#preview-locally) looks OK.

Reply:
H @sftim , I was inspecting in the preview page and looks like the png file wasn't been built together hence it cannot find the file. Is there any recommendation on this? Thank you!

Reply:
@sftim friendly ping to see if it is expected behavior(not build png file for preview) or if I did something wrong :)

Reply:
BTW sorry about the wait - I have a fairly long backlog of PRs etc where people have mentioned me, and I tend to use an oldest-first approach to find these